eval/scripts/run-finding-restatement-eval.mjs

```
@@ -0,0 +1,575 @@
+    && result.atomVerbatim
+    && result.echoOk
+    && result.quoteTargetConsistent
+    && (c.relation === 'new' || c.relation === null || c.relation === undefined)
+    && (!Array.isArray(c.targetFindingIds) || c.targetFindingIds.length === 0);
+  return result;
+}
+
+// ---------------------------------------------------------------- runner
+
+async function withTimeout(factory, ms) {
+  const controller = new AbortController();
```

ESLint

[error] 'AbortController' is not defined.

(no-undef)